R/reactives.R

```diff
@@ -1715,7 +1715,8 @@ reactivePoll <- function(intervalMillis, session, checkFunc, valueFunc) {
 
   rv <- reactiveValues(cookie = isolate(checkFunc()))
 
-  observe({
+  o <- observe({
+    cat(".")
```

Comment:
?

Reply:
Oh yeah, I should have mentioned that this is meant to be a draft PR. That's there for debugging.